validator/client/validator_propose.go

```diff
@@ -55,31 +55,31 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64, pk string) {
 		log.WithError(err).Error("Failed to get domain data from beacon node")
 		return
 	}
-	root, err := ssz.SigningRoot(b)
+	root, err := ssz.SigningRoot(block)
 	if err != nil {
 		log.WithError(err).Error("Failed to sign block")
 		return
 	}
 	signature := v.keys[pk].SecretKey.Sign(root[:], domain.SignatureDomain)
-	b.Signature = signature.Marshal()
+	block.Signature = signature.Marshal()
```

Comment:
Why change it? 'b' was fine there? didn't see any warning on that

Reply:
`b` is a vague name, figured it would be a good idea to change it.

Reply:
b is good for local variable, it's close to what go standard recommended. This is a great reference: https://talks.golang.org/2014/names.slide#6

e.g. r for reader, w for writer, etc

Reply:
Most time we use block instead of b is because b is already used as one of the package name

Reply:
Ahhh I see, I'll info it then